app/components/shared/inputs/SliderInput.vue.ts

```diff
@@ -15,6 +15,12 @@ export default class SliderInput extends BaseInput<number, ISliderMetadata> {
   @Inject() customizationService: CustomizationService;
 
   @Prop() readonly value: number;
+  @Watch('value')
+  private syncLocalValue(newVal: number) {
+    this.localValue = newVal;
+  }
```

Comment:
Is this a behavioral change?  I guess I don't fully understand why this is needed, or if it is needed, how it was working before.

Reply:
It's a behavioral change. The 2-way binding didn't work at all after some of refactoring commits. You could render an initial value or manually change a slider position